train_configs/debug_model.toml

```diff
@@ -39,6 +39,7 @@ tensor_parallel_degree = 1
 pipeline_parallel_degree = 1
 fp8_linear = ""
 compile = false
+compile_rmsnorm = true  # compile setting above should be false to use this
```

Comment:
I think we shouldn't enable it everywhere before we understand when it works and when it doens't, e.g. 2D, selective AC, etc.